### PR TITLE
add SimpleSplitCHRes kernel and test file

### PR DESCRIPTION
**Pull request by Biyss**
Refs #6910

### CODE REVIEW

File: modules/phase_field/tests/phase_field_kernels/tests [hunk 1]
```diff
@@ -60,4 +60,11 @@
     input = 'ConstructionOrder.i'
     exodiff = 'ConstructionOrder_out.e'
   [../]
+
+  [./SimpleSplitCHWRes]
+    type = 'Exodiff'
+    input = 'SimpleSplitCHWRes.i'
+    exodiff = 'SimpleSplitCHWRes_out.e'
+    max_parallel = 1
```
Comment (permcody) [inline modules/phase_field/tests/phase_field_kernels/tests]: Just curious, why the limitation here? Does it diff in parallel?

Comment (Biyss) [inline modules/phase_field/tests/phase_field_kernels/tests]: I followed the block of SplitCahnHilliard. I don't think it's really useful, but not sure about it. So I just keep it there.

Comment (permcody) [inline modules/phase_field/tests/phase_field_kernels/tests]: I see it now, this test is using the "lu" preconditioner.

Comment (Biyss) [inline modules/phase_field/tests/phase_field_kernels/tests]: Yes, exactly. Does that mean it can run in parallel, and we need to fix max_parallel to 1?

Comment (dschwen) [inline modules/phase_field/tests/phase_field_kernels/tests]: Try how it performs with asm/ilu (or just remove all petsc options)

Comment (permcody) [inline modules/phase_field/tests/phase_field_kernels/tests]: This is fine for a test, sometimes "lu" is the fastest way to get something done, but it doesn't work in parallel so you can leave it for now.

Comment (Biyss) [inline modules/phase_field/tests/phase_field_kernels/tests]: I remove them and run the test again, nothing happens...

Comment (permcody) [inline modules/phase_field/tests/phase_field_kernels/tests]: What exactly did you remove, the max_parallel? or the "lu" option in your input file? I wouldn't expect much to happen in serial. You'd have to run the test in parallel to see any differences. Stop by my cube if you have any questions.

Comment (Biyss) [inline modules/phase_field/tests/phase_field_kernels/tests]: I removed the max_parrallel and petsc options. I use -p 2 for the test, it seems fine. In this case, should I create a new pull request in next step? Thanks.

Comment (permcody) [inline modules/phase_field/tests/phase_field_kernels/tests]: No need to create a new PR - you can just amend this one.

**Comment by permcody** [comment]
This looks fine to me but @dschwen will probably need to review.

**Pull request merged**